sections/introduction.tex

```diff
@@ -2,18 +2,18 @@ \chapter{Introduction} % (fold)
 \label{cha:introduction}
 Ever since the dawn of mathematics, mathematicians has strived for faster computations while maintaining correctness.
 In our time computers have become the main target for doing extensive computations, and their CPUs are what the majority of programming languages apply to.
-CPUs are good at handling sequential instructions and moderns CPUs even exploit their multiple cores in order to attend to serial and task parallel workloads.
+CPUs are good at handling sequential instructions and modern CPUs even exploit their multiple cores in order to attend to serial and task parallel workloads.
 As applications and computations grow larger and more complex, programmers and mathematicians find themselves in need of more and more computation power.
-At the moment this demand is met mostly with faster multicored CPUs however Moore's law, which predicts an exponential growth in transistor count, thus faster computations on said CPUs, seems to be a trend coming to an end.
-Experts foresee this stagnation of Moore's Law as soon as 2020 or 2022; hence other options than CPUs need to be explored.\citep{Moore2013}
+At the moment this demand is met mostly with faster multi-cored CPUs however Moore's law, which predicts an exponential growth in transistor count, thus faster computations on said CPUs, seems to be a trend coming to an end.
+Experts foresee this stagnation of Moore's Law as soon as 2020 or 2022; hence other options than CPUs need to be explored. \citep{Moore2013}
```

Comment:
Jeg synes ikke sætningen giver kontekstuel mening. 
Vi får hurtigere computation pga. af de multiple cores, men det har jo intet med moore's lov at gøre ?
Det bør deles op tænker jeg ? 
Moore's lov stagnere, dvs. vores cores vil ikke fortsætte med at udvikle deres hastighed, så derfor sætter vi flere cores på for yderligere at øge hastigheden? Eller har jeg misforstået noget :D ?